plugins/permission-node/src/policy/types.ts

```diff
@@ -83,6 +83,6 @@ export type PolicyDecision =
 export interface PermissionPolicy {
   handle(
     request: PolicyAuthorizeRequest,
-    user?: BackstageIdentityResponse,
+    user?: BackstageUserIdentity,
```

Comment:
Just to be sure to have the discussion, this removes the access that the handler currently has to the user's Backstage token. Is that intentional and something we want/is alright? One way to potentially make this a little bit forwards compatible for now and defer any kind of decision like that is to make the second parameter `{ identity?: BackstageUserIdentity }` for now

Reply:
Ah just saw the hidden discussion above too. Thinking the `identity.identity` issue could be worked around by encouraging direct destruction into `{ identity }` for the param

Reply:
unless I've got my types mixed up here, this `BackstageIdentityResponse` coming from `auth-backend` only has one field, which is `identity: BackstageUserIdentity`. is that incorrect, and it should include the `token` as well like the type from `core-plugin-api` does? or am I using these incorrectly?

Reply:
oh although i did see the token on there when logging the identity out from the policy..either way it's a good point, the token isn't used now but might be nice to keep as is and destructure instead

Reply:
just blabbing as i work now :) destructuring here would actually be problematic, if doing `{ identity }: BackstageIdentityResponse` since the response could be undefined. options I can think of are give it a default value, or destructure within `handle` and give it an alias. or name the param `user` like it is in the `PermissionPolicy` type definition, and grab `identity` off that instead.

Reply:
[`BackstageIdentityResponse`](https://github.com/backstage/backstage/blob/9b84f1228cdb67cec912dea9cc779de5db976784/plugins/auth-backend/src/providers/types.ts#L212) extends [`BackstageSignInResult`](https://github.com/backstage/backstage/blob/9b84f1228cdb67cec912dea9cc779de5db976784/plugins/auth-backend/src/providers/types.ts#L175), which is the type the has the token

Reply:
ok reverted back to using `BackstageIdentityResponse`, and just destructured inside of the handle method of the `SimplePermissionPolicy` instead